src/Input/RadioButton/RadioButton.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import RadioButton from './RadioButton';
+
+const singleRadioButton = (
+  <RadioButton label="Full Time" name="job-type" value="full-time" />
+);
+const twoRadioButtons = (
+  <React.Fragment>
+    <RadioButton label="Full Time" name="job-type" value="full-time" />
+    <RadioButton label="Intership" name="job-type" value="intership" />
+  </React.Fragment>
+);
+const twoRadioButtonsOneChecked = (
+  <React.Fragment>
+    <RadioButton
+      label="Full Time"
+      name="job-type"
+      value="full-time"
+      checked={true}
+    />
+    <RadioButton label="Intership" name="job-type" value="intership" />
+  </React.Fragment>
+);
+
+const setupRadioButtons = () => {
+  const onChange = jest.fn(e => e.persist());
+  const { container, getByTestId } = render(
+    <React.Fragment>
+      <RadioButton
+        label="Full Time"
+        name="job-type"
+        value="full-time"
+        data-testid="radio-1"
+        onChange={onChange}
+      />
+      <RadioButton
+        label="Intership"
+        name="job-type"
+        value="internship"
+        data-testid="radio-2"
+        onChange={onChange}
+      />
+    </React.Fragment>
+  );
+  return {
+    onChange,
+    container,
```

Comment:
`container` isn't used, can consider removing it.

Reply:
Removed!